experimental/components/MsGraph/BotPlugin.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Components.MsGraph
+{
+    using Microsoft.Bot.Builder;
+    using Microsoft.Bot.Builder.Runtime.Plugins;
+    using Microsoft.BotFramework.Composer.CustomAction;
```

Comment:
Per team style convention, using statements should go above namespace declaration.